generic-db-argument-setter/examples/example-db-argument-setter-cdap-data-pipeline.json

```diff
@@ -0,0 +1,135 @@
+                        "filenameOnly": "false",
+                        "recursive": "false",
+                        "ignoreNonExistingFolders": "false",
+                        "fileEncoding": "UTF-8",
+                        "schema": "{\"type\":\"record\",\"name\":\"fileRecord\",\"fields\":[{\"name\":\"offset\",\"type\":\"long\"},{\"name\":\"body\",\"type\":\"string\"}]}"
+                    }
+                },
+                "outputSchema": "{\"type\":\"record\",\"name\":\"fileRecord\",\"fields\":[{\"name\":\"offset\",\"type\":\"long\"},{\"name\":\"body\",\"type\":\"string\"}]}",
+                "id": "File",
+                "type": "batchsource",
+                "label": "File",
+                "icon": "icon-file"
+            },
+            {
+                "name": "Trash",
+                "plugin": {
+                    "name": "Trash",
+                    "type": "batchsink",
+                    "label": "Trash",
+                    "artifact": {
+                        "name": "trash-plugin",
+                        "version": "1.2.0",
+                        "scope": "USER"
+                    },
+                    "properties": {
+                        "referenceName": "Trash"
+                    }
+                },
+                "outputSchema": [
+                    {
+                        "name": "etlSchemaBody",
+                        "schema": "{\"type\":\"record\",\"name\":\"fileRecord\",\"fields\":[{\"name\":\"offset\",\"type\":\"long\"},{\"name\":\"body\",\"type\":\"string\"}]}"
+                    }
+                ],
+                "inputSchema": [
+                    {
+                        "name": "File",
+                        "schema": "{\"type\":\"record\",\"name\":\"fileRecord\",\"fields\":[{\"name\":\"offset\",\"type\":\"long\"},{\"name\":\"body\",\"type\":\"string\"}]}"
+                    }
+                ],
+                "id": "Trash",
+                "type": "batchsink",
+                "label": "Trash",
+                "icon": "icon-TrashSink"
+            },
+            {
+                "name": "Database Argument Setter",
+                "plugin": {
+                    "name": "DatabaseArgumentSetter",
+                    "type": "action",
+                    "label": "Database Argument Setter",
+                    "artifact": {
+                        "name": "db-argument-setter",
+                        "version": "1.5.2",
+                        "scope": "USER"
+                    },
+                    "properties": {
+                        "jdbcPluginName": "mysql",
+                        "connectionString": "${connection_string}",
+                        "user": "${username}",
+                        "password": "${password}",
+                        "databaseName": "${database}",
+                        "tableName": "${table}",
+                        "argumentSelectionConditions": "${argument_selection_conditions}",
```

Comment:
line 113 and 114 shouldn't need to be macros right? Otherwise it is very hard to understand how this works?